<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saisirLesNotes</class>
 <widget class="QDialog" name="saisirLesNotes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#saisirLesNotes{
background:rgb(255, 170, 0);}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>681</width>
     <height>541</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{
background:rgb(255, 255, 255);}</string>
   </property>
   <property name="title">
    <string>REPONSE</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Numero de l'Admin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>REPONSE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Date de Reponse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NumEtud">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="NoteObtenu">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="DateDEval">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSaisir">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#buttonSaisir{
background:rgb(255, 0, 0);}</string>
    </property>
    <property name="text">
     <string>ENVOYER</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idReclam">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>L'id de Reclamtion</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
